pkg/processor/cloudevent/binary.go

```diff
@@ -91,3 +91,8 @@ func (s *Binary) GetLastInBatch() bool {
 func (s *Binary) GetOffset() int {
 	return s.event.GetOffset()
 }
+
+// GetTopic returns the topic of the event
+func (s *Binary) GetTopic() string {
```

Comment:
Isn't it enough that `wrappedEvent` will implement it?

Reply:
fixed